### ISSUE TITLE
Report a Data Problem - page layout

### ISSUE THREAD

**Issue by DeniseColbert** [FreeUKGen/FreeBMD2#397]
Suggested changes to layout

![Data Problem Report page-page-001.jpg](https://images.zenhubusercontent.com/5cab6cc818447254e461e741/4d8032d2-25f2-4d07-bd68-bea53f7a0384)

**Comment by richpomfret** [comment]
@PatReynolds to review.

**Comment by PatReynolds** [comment]
Final paragraph may need changing - seems more relevant to FreeREG.  If with (eg a marriage) the GRO index has a variant of a name and the local registerar or church register has something different,  should the alternative  - and more likely correct name -  be in a postem, not a change to the data?

**Comment by PatReynolds** [comment]
Rather than 'please find' surely 'please explain'?

Draft here: https://docs.google.com/document/d/1LvAktX3uKMeO0qcS67PtPPIMwXsUSUrlM9ZAglJggCc/edit?usp=sharing

**Comment by richpomfret** [comment]
@PatReynolds to reshare document and update based on report scan issue just discussed.

**Comment by DeniseColbert** [comment]
Sorry... Scan issue affects the Contact Us form (different issue) and not Pat's document above, which is specifically about the final paragraph of the page in question.

**Comment by DeniseColbert** [comment]
Looks good, should we run it by Barrie @PatReynolds ?

**Comment by PatReynolds** [comment]
Asked Barrie 7/2/2022

**Comment by PatReynolds** [comment]
"Personal use only" seems not to have been on FreeBMD1.  Please delete this line.

**Comment by richpomfret** [comment]
@DeniseColbert to review.

**Comment by DeniseColbert** [comment]
Reviewed. Please go ahead and implement the mock-up, but replace the final paragraph with this text:

Thank you for reporting a possible error in our transcription. [BOLD]  **We will change our transcription if we agree with your interpretation of the source document we transcribed** [BOLD]. Please let us know the image number, if you found it on FreeBMD.

If you have found a clearer image which we do not have, please let us know where.  If there is no restriction on the sharing of the image, you can add a copy to your request. It is your responsibility to check that you are not breaching the terms and conditions of websites.

**Comment by richardofsussex** [comment]
I think we need to think this through from scratch, then carry out major surgery on this page. Here's my take on this issue:

There are two possible reasons why a user might want to report a 'data problem'. One is that they think the source may have been mis-transcribed; the other is that they think they have better data than what we are offering.

I think that we should _only_ be dealing with the first of these cases. As noted above, this is the only situation in which we would be prepared to update our actual data; any other data enhancement should go into a postem. And I would further argue that putting useful additional information into postems is a pointless activity, since it will not be findable by anyone. Our data is an index into a set of registers; it is _not_ a set of records about specific people. Therefore 'enhancing' it with additional unstructured information is detracting from its core purpose, not enhancing it.

That being the case, the only source the user might have, which would be relevant and useful, would be a better scan of the _same page_ of the index. (This came up in the FreeBMD SG meeting yesterday - Alan said that most second keying has, in fact, been done exactly because better scans became available.) So we shouldn't be asking for scans of any other source.

**Comment by richardofsussex** [comment]
On second reading, I can see that a great deal of thought has gone into this page, and that it broadly does what I was suggesting it should do, i.e. discourage users from submitting unnecessary correction/addition requests. I also now realise that it is a Refinery page, which complicates the process of updating it.

What I would suggest is that we make it clearer at the start of the page that many of the issues that users might think of reporting are either not errors in our view, or they are not relevant to a database which is a transcription of the GRO Index. My next comment is a suggested wording for the page, up to "Select your query".

The whole of the existing text ("Thank you for reporting...complete the form below.") could appear _after_ the user has read our explanation and then clicked on the "Yes, I have read the explanation ..." checkbox. It is useful and relevant if they are actually going to submit a report.

It is likely that the wording of some of the explanations could usefully be updated, in the light of the changed content at the start of the page. I suggest we deal with that as a second step.

**Comment by richardofsussex** [comment]
Heading: Errors and omissions in the GRO Index data

The data in the FreeBMD web site is transcribed by our volunteers from scans of pages in the General Register Office indexes. These indexes were created by copying selected information from certificates, pooling it centrally, and creating an index by hand. The format of these indexes has changed over time, from handwritten pages, to typed pages, then to computer printout. The quality of the scans we have been given access to (often scans of fiche or microfilm, which itself a copy of the source page) has also varied - some are very hard to read.

In the course of all this copying of data, it is inevitable that mistakes and omissions will occur. Our goal is to ensure that, as far as possible, our data is a faithful transcription of what is actually present in the GRO index. We recognize, and accept, that the index itself may be wrong. We do provide a mechanism for recording additional information about an index entry (a "postem"), but you should be aware that anything recorded in a postem will not be searchable, and will only be seen by others if they happen to find the entry by other means.

Please select the option below which best describes your reason for thinking the data in this entry is wrong, and an explanation will appear, advising what (if anything) you should report to us.

**Comment by richardofsussex** [comment]
This is on test3: https://test3.freebmd.org.uk/contacts/237864549/report_error?locale=en&query=66c45398bd30f2e961965732. Can we agree it and move on (e.g. to #398)?

**Comment by KristinaGadzhieva** [comment]
Thank you very much, Richard.
@richardofsussex @nurifreembd 
Some suggestions:
~1.	Provide a link to the Help page section related to how to create Postems~
2.     Add a link to the Help section (on how to add postems) within the explanation sections of the "Some data is missing" and "A query about the data that is presented".
~3.	Can the word ‘entry’ be replaced with ‘record’ ? Perhaps, it would be clearer for users.~
4.	Change the name of the button from 'Report Error' to 'Submit Report'
~5.	How about removing 'Give Us Feedback' from this page to avoid confusing users?~
6.	Remove tooltips/info icons

**Comment by DeniseColbert** [comment]
Re. @KristinaGadzhieva's point 6. tooltips/info icons are being decommissioned, so these need removing altogether, as I understand. There are none on the search page anymore. (I can't see any magenta ones either)

**Comment by KristinaGadzhieva** [comment]
@DeniseColbert Right, I remember that we removed info icons, but we still have one on the Search page. (Mother's Surname).

**Comment by DeniseColbert** [comment]
Ah! I didn't check the different forms for record types!

@AlOneill can you remember if/why we consciously decided to keep the tooltip on Mother's Surname?

**Comment by AlOneill** [comment]
@DeniseColbert 
Not exactly, but I guess it was because users were not sure _which_ surname we were asking for?

**Comment by KristinaGadzhieva** [comment]
If tool tips are not appropriate from an accessibility point of view, why would we make any exceptions (for Mother's Surname)? I think it's just a mistake. We discussed removing all info icons.

**Comment by DeniseColbert** [comment]
Yes @KristinaGadzhieva that's what I was trying to ascertain. Either way, I think making the title Mother's Maiden Name should make it (somewhat) clearer so we can remove the tooltip.

**Comment by richardofsussex** [comment]
Like this? : -
![image](https://github.com/user-attachments/assets/b63526f8-279a-4f4a-bf70-f7fd5e39d61d)

**Comment by richardofsussex** [comment]
Code is in branch `fb_397_rbl_2`. @Vino-S please push to test3.

**Comment by Vino-S** [comment]
@richardofsussex test3 updated

**Comment by richardofsussex** [comment]
I've done 3 and 4. As regards 1 and 2, I don't see where there is any help text about creating, or using, postems, to link to. As regards 5, the 'Give Us Feedback' link is effectively part of the standard page footer. It's in app/views/layouts/application.html.erb. @Vino-S may know, but I don't, how to test which page you're on in order to suppress this link. The small amount I _have_ done is is `fb_397_rbl_3` and could be added to test3, please @Vino-S.

**Comment by KristinaGadzhieva** [comment]
Thank you, Richard. 
1 and 2 can be added after completing the Help page.

**Comment by nurifreembd** [comment]
Hello Kristina,

re point 1, create postem, what happens to the created postem? should the
user see a list of postem from other users? should there be a link to
postem on the menu on the right for example create a postem and view
postem(s)?

**Comment by KristinaGadzhieva** [comment]
@richardofsussex Unfortunately, some of the changes seem to have disappeared from Beta. Richard, could you please take a look at points 2, 4, and 6?
- [ ] 2.     Add a link to the Help section (on how to add postems) within the explanation sections of the "Some data is missing" and "A query about the data that is presented".
- [x] 4.	Change the name of the button from 'Report Error' to 'Submit Report'
- [x] 6.	Remove tooltips/info icons

**Comment by KristinaGadzhieva** [comment]
High priority, for testing

**Comment by richardofsussex** [comment]
Point 2: because the Help section on how to add postems is now hidden by default, it is not possible to link to it directly (unless @Vino-S knows otherwise ...). All we could do is provide a link to the Help 'home page'.

**Comment by richardofsussex** [comment]
I've done 4 and 6.

**Comment by KristinaGadzhieva** [comment]
@richardofsussex Thanks, Richard. Let's fix point 2 before Beta release.

**Comment by richardofsussex** [comment]
Alison has already pointed out accessibility issues with our new 'help switching' approach (#549). Point 2 is in exactly the same area. We need to make a decision as to whether we persevere with this approach, or if we should revert to having separate help pages, each of which is addressable in the normal way.

**Comment by DeniseColbert** [comment]
@Vino-S  to add an ID to Postem section of Help and link to it on blank checkbox above

**Comment by Vino-S** [comment]
Changes deployed to LIVE

**Comment by DeniseColbert** [comment]
Done closing
